robot/demos/RemoteArmControl/comms.sh

```diff
@@ -0,0 +1,2 @@
+terminator -T "ServerListenerWindow" -e "./ServerListenerLocal.py" &
```

Comment:
This script should start with a shebang.
Adding `#!/usr/bin/env bash` as the first line will do the trick.